docs/source/user-guide/tabular/custom_check_templates.rst

```diff
@@ -0,0 +1,185 @@
+.. _custom_check_templates:
+
+======================
+Custom Check Templates
+======================
+
+This page supplies templates for the different types of custom checks that you can create using the deepchecks package.
+For more information on custom checks, please see the
+:doc:`Custom Check Guide. </user-guide/tabular/tutorials/plot_add_a_custom_check>`
+
+
+
+Templates:
+
+* `Single Dataset Check <#single-dataset-check>`__
+* `Train Test Check <#train-test-check>`__
+* `Model Only Check <#model-only-check>`__
+
+
+Single Dataset Check
+--------------------------
+
```

Comment:
Can we add one/two-sentence descriptions for each of these?
(extracted from api reference or written directly here)
For each of the three types